string name                          # object's name
geometry_msgs/Quaternion direction   # new direction
--- 
